app/services/data/get-arms-totals.js

```diff
@@ -0,0 +1,36 @@
+const config = require('../../../config')
+const knexConfig = require('../../../knexfile').app
+const knex = require('knex')(knexConfig)
+
+module.exports = function (omKey, teamCode) {
+  var whereObject = {
+    'assessment_staff_key': omKey,
+    'assessmentent_team_key': teamCode // typo in extract
+  }
+
+  return knex.count('sentence_type as count').select('sentence_type')
+          .from(`${config.DB_STG_SCHEMA}.arms`)
+          .where(whereObject) // .andWhere(disposal_or_release_date is no more than 6 weeks ago)
+          .groupBy('sentence_type')
+  .then(function (results) {
+    var armsTotals = {
+      community: 0,
+      license: 0
+    }
+
+    results.forEach(function (row) {
+      switch (row.sentence_type) {
+        case 'Community':
```

Comment:
We have generally been using all caps on these keywords. Are we sure this is the case they will be coming in in the extract?

Reply:
It was in the sample extract that Kristy sent to Tom. I'll move these to `const` so if they're different when the file lands we can easily change them.